pkg/apis/minio.min.io/v1/types.go

```diff
@@ -132,6 +132,19 @@ type TenantSpec struct {
 	// SideCars a list of containers to run as sidecars along every MinIO Pod on every pool
 	// +optional
 	SideCars *SideCars `json:"sideCars,omitempty"`
+	// ExposeServices tells operator whether to expose the MinIO service and/or the Console Service
+	// +optional
+	ExposeServices *ExposeServices `json:"exposeServices,omitempty"`
+}
+
+// ExposeServices tells operator whether to expose the services for MinIO and Console
+type ExposeServices struct {
```

Comment:
Since the user needs to explicitly list only those services that need to be accessed outside k8s, wouldn't it be simpler to have it as a list like so?
```yaml
exposeServices:
  - minio
  - console
```
If a user changes their mind about exposing the console service, they can update the spec to look like,
```yaml
exposeServices:
   - minio
```

Reply:
I like the idea, I was also thinking on using an enum so that in the future an ingress controller could be configured from here